Userland/Libraries/LibJS/Runtime/RegExpObject.h

```diff
@@ -37,6 +37,17 @@ class RegExpObject : public Object {
         | regex::ECMAScriptFlags::BrowserExtended
     };
 
+    enum class Flags {
```

Comment:
You could use `AK_ENUM_BITWISE_OPERATORS` to get all the bitwise operations and a `has_flag` helper for free, then you don't need to do all the conversions to `u32`

Reply:
Yes sir, good idea!